test/TestProjects/FirstTest-TypeSpec/src/Generated/Models/AnonymousBodyRequest.cs

```diff
@@ -1,195 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-// <auto-generated/>
-
-#nullable disable
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Azure.Core;
-
-namespace FirstTestTypeSpec.Models
-{
-    /// <summary> The AnonymousBodyRequest. </summary>
-    internal partial class AnonymousBodyRequest
```

Comment:
Is this expected?

Reply:
I believe this is expected.
One of the TCGC's change is that it does not always generate a XXRequest model for spread body. It will try to reuse the model